src/main/java/fr/insee/pogues/configuration/StaticResourcesForFOPConfig.java

```diff
@@ -20,10 +20,16 @@
 @Component
 public class StaticResourcesForFOPConfig {
 
-    final static Logger logger = LogManager.getLogger(StaticResourcesForFOPConfig.class);
+    final static Logger LOG = LogManager.getLogger(StaticResourcesForFOPConfig.class);
```

Comment:
this boilerplate could be replaced with a lombok annotation at class level like `@Slf4j`

Reply:
You have an example somewhere of usage `@Slf4j` annotation ?
Thanks

Reply:
I found this : https://medium.com/@AlexanderObregon/enhancing-logging-with-log-and-slf4j-in-spring-boot-applications-f7e70c6e4cc7

Perfect